testgrid/config.yaml

```diff
@@ -7077,6 +7080,9 @@ dashboards:
     base_options: width=10
     alert_options:
       alert_mail_to_addresses: bentheelder+alerts@google.com
+  - name: pull-kubernetes-e2e-aks-engine-azure
```

Comment:
last nit I think (😅!): this should be in `presubmits-kubernetes-nonblocking` below